private/private.go

```diff
@@ -63,17 +63,21 @@ func MustNewPrivateTxManager(cfgPath string) PrivateTransactionManager {
 		panic(fmt.Sprintf("unable to read %s due to %s", cfgPath, err))
 	}
 
+	ptm, err := newPrivateTxManager(cfg)
+	if err != nil {
+		panic(fmt.Sprintf("unable to connect to private tx manager using %s due to %s", cfgPath, err))
+	}
+	return ptm
+}
+
+func newPrivateTxManager(cfg engine.Config) (PrivateTransactionManager, error) {
```

Comment:
Why not exporting the function?

Also I think `NewPrivateTxManager(cfgPath string)` could take the config path as an argument possibly returning an extended error using `fmt.Errorf(...)` so in the end, `MustNewPrivateTxManager(cfgPath string)` only handles the panic behavior:

```go
func MustNewPrivateTxManager(cfgPath string) PrivateTransactionManager {
    if ptm, err := NewPrivateTxManager(cfgPath string); err != nil {
        panic(err)
    }
    return ptm
}
```

Reply:
The function is only used locally, hence there's no reason to export it.
I feel it's cleaner as implemented, splitting the load config from the txn manager creation, so I prefer to leave it as is.